src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/VideoLinkBookingCreateRequest.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.Valid
+import jakarta.validation.constraints.AssertTrue
+import jakarta.validation.constraints.Future
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotEmpty
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.net.URI
+import java.time.LocalDate
+import java.time.LocalTime
+
+data class VideoLinkBookingCreateRequest(
+
+  @field:NotNull(message = "The video link booking type is mandatory")
+  @Schema(description = "The booking type", example = "COURT")
+  val bookingType: BookingType?,
+
+  @field:Valid
+  @field:NotEmpty(message = "At least one prisoner must be supplied for a video link booking")
+  @Schema(description = "The prisoner or prisoners associated with the video link booking")
+  val prisoners: List<PrisonerDetails>,
+
+  @field:Size(max = 400, message = "Comments for the video link booking cannot not exceed {max} characters")
+  @Schema(description = "Free text comments for the video link booking", example = "Waiting to hear on legal representation")
+  val comments: String?,
+)
+
+enum class BookingType {
+  COURT,
+  PROBATION,
+}
+
+data class PrisonerDetails(
+
+  @field:NotEmpty(message = "Prison code is mandatory")
+  @field:Size(max = 3, message = "Prison code should not exceed {max} characters")
+  @Schema(description = "The prison code for the prisoner", example = "PVI")
+  val prisonCode: String?,
```

Comment:
I think some prison codes are 6 max - there is a test prison called ZZTEST? Something like that, but fine for now - we don't want to use this test prison anyway!